test/ffmpeg-vaapi/transcode/avc.py

```diff
@@ -0,0 +1,69 @@
+###
+### Copyright (C) 2018-2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ..util import *
+from .transcoder import TranscoderTest
+
+spec = load_test_spec("avc", "transcode")
+@platform_tags(set(AVC_DECODE_PLATFORMS)&set(AVC_ENCODE_PLATFORMS))
+class AVC(TranscoderTest):
+  @slash.parametrize(*gen_transcode_parameters(spec, "avc"))
+  def test_avc(self, case, ttype):
```

Comment:
I don't think ttype should be a test parameter.  Also, if ttype was "hwsw", then the platform_tags at the class level are wrong.  Perhaps get rid of ttype and use ```def test_hwhw(...)``` instead with platform_tags at the function level.